charts/DiscreteBarTransform.ts

```diff
@@ -41,16 +35,14 @@ export class DiscreteBarTransform implements IChartTransform {
         return this.chart.data.filledDimensions.filter(d => d.property === "y")
     }
 
-    @computed get targetYear(): number {
-        const maxYear = this.chart.timeDomain[1]
-        if (this.primaryDimensions.length === 0) return 1900
-
-        const yearsUniq = flatten(
-            this.primaryDimensions.map(dim => dim.variable.yearsUniq)
+    @computed get timelineYears(): Time[] {
+        return sortedUniq(
+            sortBy(
+                flatten(
+                    this.primaryDimensions.map(dim => dim.variable.yearsUniq)
+                )
+            )
```

Comment:
nit: You could be more consistent with how these are implemented in all the transforms :)
Sometimes it's `map, flatten, sortBy, sortedUniq`, other times its `map, union, sortBy, sortedUniq` (`union` already does `uniq`!). Btw, `map + flatten` is just `flatMap`.